charts/alertmanager-snmp-notifier/values.yaml

```diff
@@ -0,0 +1,185 @@
+  #     - Alert: {{ $alert.Labels.alertname }}
+  #       Summary: {{ $alert.Annotations.summary }}
+  #       Description: {{ $alert.Annotations.description }}
+  #     {{ end }}
+  #     {{ end }}
+  #     {{ else -}}
+  #     Status: OK
+  #     {{- end -}}
+
+  #   extraFields:
+  #     - template: |
+  #         {{- if .Alerts -}}
+  #         Status: NOK
+  #         {{- else -}}
+  #         Status: OK
+  #         {{- end -}}
+  #     - template: |
+  #         This is a constant
+
+## Security context to be added to snmp-notifier pods
+securityContext:
+  {}
+  # fsGroup: 1000
+  # runAsUser: 1000
+  # runAsNonRoot: true
+
+## Security context to be added to snmp-notifier containers
+containerSecurityContext:
+  runAsNonRoot: true
+  runAsUser: 1000
+  readOnlyRootFilesystem: true
+
+## Additional labels to add to all resources
+customLabels:
+  {}
+  # app: snmp-notifier
+
+extraConfigmapMounts:
+  []
+  # - name: snmp-notifier-configmap
+  #   mountPath: /run/secrets/snmp-notifier
+  #   subPath: snmp.yaml # (optional)
+  #   configMap: snmp-notifier-configmap-configmap
+  #   readOnly: true
+  #   defaultMode: 420
+
+## Additional secret mounts
+# Defines additional mounts with secrets. Secrets must be manually created in the namespace.
+extraSecretMounts:
+  []
+  # - name: secret-files
+  #   mountPath: /run/secrets/snmp-notifier
+  #   secretName: snmp-notifier-secret-files
+  #   readOnly: true
+  #   defaultMode: 420
+
+## For RBAC support:
+rbac:
+  # Specifies whether RBAC resources should be created
+  create: true
+
+serviceAccount:
+  # Specifies whether a ServiceAccount should be created
+  create: true
+
+  # The name of the ServiceAccount to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+
+resources:
+  {}
+  # limits:
+  #   memory: 300Mi
+  # requests:
+  #   memory: 50Mi
+
+livenessProbe:
+  httpGet:
+    path: /health
+    port: http
+readinessProbe:
+  httpGet:
+    path: /health
+    port: http
+
+service:
+  annotations: {}
+  type: ClusterIP
+  port: 9464
+
+## An Ingress resource can provide name-based virtual hosting and TLS
+## termination among other things for CouchDB deployments which are accessed
+## from outside the Kubernetes cluster.
+## ref: https://kubernetes.io/docs/concepts/services-networking/ingress/
+ingress:
+  enabled: false
+  hosts:
+    []
+    # - chart-example.local
+  annotations:
+    {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  tls:
+    []
+    # Secrets must be manually created in the namespace.
+    # - secretName: chart-example-tls
+    #   hosts:
+    #     - chart-example.local
+
+podAnnotations: {}
+
+replicas: 1
+
+# Enable this if you're using https://github.com/coreos/prometheus-operator
+serviceMonitor:
+  # When set true then use a ServiceMonitor to configure scraping
+  enabled: false
+  # Set the namespace the ServiceMonitor should be deployed
```

Comment:
values dont have to be commented out if enbaled is false anyway.

Reply:
Indeed, this is fixed.